### PR TITLE
issue/113/demo_dc2

### PR DESCRIPTION
**Pull request by maho3**
adding changes from Issue/110/refactor_clmm_api. added new Cluster object definition as well as a use-case ipynb and loading/centering scripts

### CODE REVIEW

File: clmm/pipeline/load_from_DC2.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+
```
Comment (matthewkirby) [inline clmm/pipeline/load_from_DC2.py]: Put this file into clmm/desc/

**Comment by maho3** [comment]
Comments from the all-hands meeting:
-[ ] .gals --> .gal_cat to make it more obvious
-[ ] .id --> .cl_id to avoid overwriting python's id definition
-[ ] Someone else should copy old galaxycluster.py unit tests to test the new format

**Comment by matthewkirby** [comment]
The GalaxyCluster object has been merged into master. This branch still contains the GCR loading scripts and examples.

**Comment by aimalz** [comment]
Does this PR close #93 in addition to #110?  (And can the PR be closed?)

**Comment by maho3** [comment]
In regards to #93 , the GalaxyCluster object does not yet have a str or repr function, so that issue is not yet closed.
This PR has closed #110 but contains code which addresses #133

**Comment by aimalz** [comment]
It looks like the changes @matthewkirby requested have been made, so I guess this PR's ready to merge, right?

**Comment by maho3** [comment]
Adding documentation to the load_from_dc2.py and checking that it works with any and all changes to master. I'll ping ya when this is good for another review.

**Comment by aimalz** [comment]
@maho3 Did we merge this prematurely?

**Comment by maho3** [comment]
@aimalz @cavestruz Ah yes you did, I was working on documentation and making sure it works with master

**Pull request merged**